package/usr/local/bin/hassbian-config

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
-
 SUITE_INSTALL_DIR=/opt/hassbian/suites
 SUITE_INSTALL_DB=/srv/homeassistant/hassbian/suite-states
 SUITE_INSTALL_DB_LOCK=/srv/homeassistant/hassbian/suite-states.lock
+SUITE_CONTROL_DIR=/opt/hassbian/control
```

Comment:
View the previous comment.

Reply:
Ups, removed all referances to old state functions now

Reply:
Moved control dir to srv/homeassistant/hassbian/control